source/common/upstream/load_balancer_impl.h

```diff
@@ -707,11 +707,19 @@ class LeastRequestLoadBalancer : public EdfLoadBalancerBase {
 
     double host_weight = static_cast<double>(host.weight());
 
+    // If the value of active requests is the max value, adding +1 will overflow
+    // it and cause a divide by zero. This won't happen in normal cases but stops
+    // failing fuzz tests
+    auto active_request_value =
```

Comment:
nit: const, and use an explicit type for easier understanding.

Reply:
Done